test/e2e_node/e2e_node_suite_test.go

```diff
@@ -205,8 +208,13 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// Reference common test to make the import valid.
 	commontest.CurrentSuite = commontest.NodeE2E
 
-	return nil
-}, func([]byte) {
+	// ginkgo would spawn multiple processes to run tests.
+	// Since the bearer token is generated randomly at run time,
+	// we need to distribute the bearer token to other processes to make them use the same token.
+	klog.Infof("Distribute bearer token: %s", framework.TestContext.BearerToken)
```

Comment:
i think we can omit this klog.Infof() line.

Reply:
fixed